cmake/modules/FindLUG.cmake

```diff
@@ -50,6 +50,7 @@ endif()
 set(FIND_LUG_PATHS
     ${LUG_ROOT}
     $ENV{LUG_ROOT}
+    ${ANDROID_NDK}/sources/lugdunum
```

Comment:
didn't we did this together ? That was the problem why FingLug failed when invoked by Android Studio